frontend/src/main/java/olumofe/raphael/frontend/CarController.java

```diff
@@ -0,0 +1,57 @@
+package olumofe.raphael.frontend;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.*;
+import starter.project.domain.Car;
+
+import java.util.*;
+
+@RestController
+@Component
+public class CarController {
+
+
+    private List<Car> carNewList = new ArrayList<>();
+
+    @GetMapping(path ="/car/{carId}", consumes = {"application/json"})
+    @ResponseBody
+    public Car getCars(@PathVariable("carId") int carId) {
+
+        for (Car car : carNewList) {
+            if (car.getId() == (carId)) {
```

Comment:
If the ID is stored at the Car level, then you should think about adding a better way to access the data. Perhaps a Map<CarId, Car> would be more suitable.